packages/expo/src/generators/application/lib/normalize-options.ts

```diff
@@ -41,8 +45,34 @@ export async function normalizeOptions(
     ? options.tags.split(',').map((s) => s.trim())
     : [];
   const rootProject = appProjectRoot === '.';
+
+  let e2eWebServerTarget = 'serve';
```

Comment:
@xiongemi Could make sure some of the assumptions for `expo` projects are correct here?

- default `webServerTarget`
- default `port` the app gets served on